OpenEdX/AppDelegate.swift

```diff
@@ -125,4 +137,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window?.rootViewController = RouteController()
     }
     
+    // Push Notifications
+    func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
+        if let pushManager = Container.shared.resolve(PushNotificationsManager.self) {
+            pushManager.didRegisterForRemoteNotificationsWithDeviceToken(deviceToken: deviceToken)
+        }
+    }
+    func application(_ application: UIApplication, didFailToRegisterForRemoteNotificationsWithError error: Error) {
+        if let pushManager = Container.shared.resolve(PushNotificationsManager.self) {
```

Comment:
How about using guard let here and same for the following push notifications methods?

Reply:
It would be nice if we can follow the same pattern for `didReceiveRemoteNotification` and `configureDeepLinkServices`.